extern/storage-sealing/fsm.go

```diff
@@ -19,7 +19,13 @@ import (
 func (m *Sealing) Plan(events []statemachine.Event, user interface{}) (interface{}, uint64, error) {
 	next, processed, err := m.plan(events, user.(*SectorInfo))
 	if err != nil || next == nil {
-		return nil, processed, err
+		l := Log{
+			Timestamp: uint64(time.Now().Unix()),
+			Message:   fmt.Sprintf("state machine error: %s", err),
+			Kind:      fmt.Sprintf("error;%T", err),
```

Comment:
```suggestion
			Kind:      fmt.Sprintf("error:%T", err),
```
Just guessing that this may be a typo.

Reply:
It's not a typo, the event logger is also using `;`